src/containers/editor/graph/Node.tsx

```diff
@@ -32,7 +43,11 @@ export const ObjectNode = memo(({ id, data }: NodeProps<NodeWithData>) => {
   return (
     <>
       {data.toolbarVisible && <Toolbar id={id} />}
-      <div className="nodrag nopan graph-node cursor-default" style={data.style}>
+      <div
+        className={cn("nodrag nopan graph-node cursor-default", {
+          "opacity-50": isSiblingsFolded,
```

Comment:
I think you should add opacity on `SourceHandle` rather than here, and you need to use the `hidden` field of the target node.

Reply:
My bad I misunderstood it. I modified to change the transparency of source handles.